src/auth/base.ts

```diff
@@ -1,15 +1,11 @@
-import { ConnectOptions, Credential } from "../types.ts";
 import { WireProtocol } from "../protocol/mod.ts";
-import { Document } from "../../deps.ts";
+import { ConnectOptions, Credential, Document } from "../types.ts";
 
 export abstract class AuthPlugin {
-  abstract prepare(
-    authContext: AuthContext,
-  ): Document;
-  abstract auth(
-    authContext: AuthContext,
-  ): Document;
+  abstract prepare(authContext: AuthContext): Document;
```

Comment:
Does this follow deno fmt? we are using deno fmt

Reply:
I believe it does but I'll run deno fmt again to make sure

Reply:
this is what deno fmt output when i ran it manually so it seems right